ebay-frontend/src/app/upcoming-events-screen/upcoming-events-screen.component.html

```diff
@@ -19,4 +19,4 @@
 
 </mat-card>
 
-<button mat-fab color=primary class="createEventButton"> <mat-icon>add</mat-icon> </button>
+<a routerLink="/create-events" routerLinkActive=""><button mat-fab color=primary class="createEventButton"> <mat-icon>add</mat-icon> </button></a>
```

Comment:
You shouldn't need to surround the button with an `a` tag. You should be able to put `routerLink` directly on the button. Also, you shouldn't need the `routerLinkActive` unless you're going to enable/disable it.